samples/mpu_test/README.rst

```diff
@@ -46,6 +46,13 @@ single thread: :file:`prj_single.conf`:
 
    $ make BOARD=v2m_beetle CONF_FILE=prj_single.conf run
 
+To build the version that allows flash write, use the supplied configuration
+file: :file:`prj_allow_flash_write.conf`:
```

Comment:
no such file  (there are prj_allow_flash_write_mcux.conf and prj_allow_flash_write_stm32.conf).  Maybe use the same notation you use in the example below prj_allow_flash_write_<x>.conf and tell them what the <x> means.